iOS-Sample/Hello-World.xcodeproj/project.pbxproj

```diff
@@ -479,12 +559,11 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = C2DFB87770B49E87D49DFD8A /* Pods-Hello-World.release.xcconfig */;
 			buildSettings = {
-				ARCHS = "$(ARCHS_STANDARD)";
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = D3CUYFAV98;
```

Comment:
Could you please remove your development profile? When I run the repo, it shows up as this: https://cl.ly/3C3m2E0v1f18

Reply:
Done